src/features/wallet/root/useActivityItem.tsx

```diff
@@ -207,71 +209,85 @@ const useActivityItem = (address: string) => {
       ) {
-        return formatAmount(true, item.fee)
+        return formatAmount('-', item.fee)
       }
 
       if (item instanceof PaymentV1 || item instanceof PaymentV2) {
         if (address !== item.payer) return ''
-        return formatAmount(true, item.fee)
+        return formatAmount('-', item.fee)
       }
 
       const pendingTxn = item as PendingTransaction
       if (pendingTxn.txn !== undefined) {
-        return formatAmount(true, pendingTxn.txn.fee)
+        return formatAmount('-', pendingTxn.txn.fee)
       }
 
-      return formatAmount(true, (item as AddGatewayV1).fee)
+      return formatAmount('-', (item as AddGatewayV1).fee)
     },
     [address, isSelling],
   )
 
   const amount = useCallback(
     (item: AnyTransaction | PendingTransaction) => {
       if (item instanceof TransferHotspotV1) {
-        return formatAmount(!isSelling(item), item.amountToSeller)
+        return formatAmount(isSelling(item) ? '+' : '-', item.amountToSeller)
       }
       if (item instanceof AssertLocationV1 || item instanceof AddGatewayV1) {
-        return formatAmount(true, item.stakingFee)
+        return formatAmount('-', item.stakingFee)
       }
       if (item instanceof TokenBurnV1) {
-        return formatAmount(true, item.amount)
+        return formatAmount('-', item.amount)
       }
       if (item instanceof RewardsV1) {
-        return formatAmount(false, item.totalAmount)
+        return formatAmount('+', item.totalAmount)
       }
       if (item instanceof PaymentV1) {
-        return formatAmount(item.payer === address, item.amount)
+        return formatAmount(item.payer === address ? '-' : '+', item.amount)
       }
       if (item instanceof PaymentV2) {
         if (item.payer === address) {
-          return formatAmount(true, item.totalAmount)
+          return formatAmount('-', item.totalAmount)
         }
 
         const payment = item.payments.find((p) => p.payee === address)
-        return formatAmount(false, payment?.amount)
+        return formatAmount('+', payment?.amount)
       }
 
       const pendingTxn = item as PendingTransaction
       if (pendingTxn.txn !== undefined) {
-        if (pendingTxn.txn.type === 'add_gateway_v1') {
-          return formatAmount(true, (pendingTxn.txn as AddGatewayV1).stakingFee)
+        if (pendingTxn.type === 'add_gateway_v1') {
+          return formatAmount('-', (pendingTxn.txn as AddGatewayV1).stakingFee)
         }
-        if (pendingTxn.txn.type === 'assert_location_v1') {
+        if (pendingTxn.type === 'assert_location_v1') {
           return formatAmount(
-            true,
+            '-',
             (pendingTxn.txn as AssertLocationV1).stakingFee,
           )
         }
+        if (pendingTxn.type === 'payment_v2') {
+          const paymentV2 = pendingTxn.txn as PaymentV2
+          if (paymentV2.payer === address) {
+            const sum = paymentV2.payments.reduce(
+              (a, b) => a + b.amount.floatBalance,
```

Comment:
we should be able to use the balance arithmetic functions here to keep it as a balance object. that way there's no float rounding issues, and you probably wouldn't need `postfix` then either.

https://github.com/helium/helium-js/blob/master/packages/currency/src/Balance.ts#L69

Reply:
Actually, can I just do this?
```
 if (paymentV2.payer === address) {
   return formatAmount('-', paymentV2.totalAmount)
```

Reply:
nope, nevermind, I can't

Reply:
Wait, yes, I can. Fixed up the pending txn to have this.